web/src/components/molecules/Schema/SchemaMenu.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from "@emotion/styled";
+import React from "react";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Menu from "@reearth-cms/components/atoms/Menu";
+import { Model } from "@reearth-cms/components/molecules/Schema/types";
+
+export interface Props {
+  defaultSelectedKeys?: string[];
+}
+
+export interface Props {
+  defaultSelectedKeys?: string[];
+  models?: Model[];
+  handleModalOpen: () => void;
+  selectModel: (modelId: string) => void;
+}
+
+const SchemaMenu: React.FC<Props> = ({
+  defaultSelectedKeys,
+  models,
+  handleModalOpen,
+  selectModel,
+}) => {
+  const onClick = (e: any) => {
+    selectModel(e.key);
+  };
+
+  return (
+    <SchemaStyledMenu>
+      <SchemaStyledTitle>Schema</SchemaStyledTitle>
+      <SchemaAction>
+        <SchemaStyledMenuTitle>Models</SchemaStyledMenuTitle>
+        <SchemaAddButton
+          onClick={handleModalOpen}
+          color="#1890FF"
```

Comment:
You have the color set here and below in the styled components. Pick one and remove the other